Logger/FileLogger.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+
+namespace Logger;
+    public class FileLogger(string filePath) : BaseLogger
+    {
+        private LogLevel _logLevel;
+        private string _message = "No message given";
+        private string _className1 = "No class name given";
+        private string _filePath = filePath;
+
+        public override string ClassName
+        {
+            get { return _className1; }
+            set { _className1 = value; }
+        }
+
+        public string FilePath { get => _filePath; set => _filePath = value; }
+        public string Message { get => _message; set => _message = value; }
+        public LogLevel LogLevel1 { get => _logLevel; set => _logLevel = value; }
```

Comment:
Why is this named with the `1` suffix?
```suggestion
        public LogLevel LogLevel { get => _logLevel; set => _logLevel = value; }
```